capstone/src/main/webapp/cover.css

```diff
@@ -1,7 +1,6 @@
 /* Base structure */
 html, body {
   height: 100%;
-  background-image: url(/assets/Pattern-Randomized.svg);
```

Comment:
Just out of curiosity, what happens if we keep this in? The white background might be a bit bland

Reply:
It'll look like this. I was going for the style on support.perspectiveapi.com, but happy to meet about style preferences
![image](https://user-images.githubusercontent.com/65875732/88573437-bb606e00-d00e-11ea-8402-2e8f8bb8dc32.png)

Reply:
Ah, I see. I think the no-background looks better now, but maybe we can make horizontal divisions of light gray like support.perspectiveapi.com at some point.